src/components/combobox/combobox.tsx

```diff
@@ -419,14 +429,16 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
         this.nextChip();
         break;
       case "ArrowUp":
-        event.preventDefault();
-        this.active = true;
         this.shiftActiveItemIndex(-1);
+        if (!this.inViewport()) {
+          this.el.scrollIntoView();
+        }
         break;
       case "ArrowDown":
-        event.preventDefault();
-        this.active = true;
```

Comment:
I thought ⬇️ would:

1. when closed, open the menu
2. when open, navigate to the next item